h2o-genmodel/src/main/java/hex/genmodel/easy/EnumLimitedEncoder.java

```diff
@@ -0,0 +1,57 @@
+package hex.genmodel.easy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EnumLimitedEncoder implements CategoricalEncoder {
```

Comment:
this class is a great candidate for a unit test

Reply:
added tests for all new classes